front-end/src/components/SignIn.js

```diff
@@ -49,17 +48,24 @@ const SignIn = (props) => {
       },
     };
 
-    axios.post(`${process.env.REACT_APP_BACKEND_SERVICE}/login`, loginCreds, config).then((res) => {
-      if (res.status === 200) {
-        console.log('SUCCESS');
-        props.onAuthChange(true);
-      } else if (res.status === 401) {
-        console.log('Invalid credentials');
-        props.signInAttempt(true);
-      } else {
-        console.log('Issue with Authentication server');
-      }
-    });
+    axios
+      .post(`${process.env.REACT_APP_BACKEND_SERVICE}/login`, loginCreds, config)
+      .then((res) => {
+        if (res.status === 200) {
+          console.log('Login SUCCESS');
+          props.onAuthChange(true);
+          props.setUserId(res.data);
+        }
+      })
+      .catch((err) => {
+        if (err.response.status === 401) {
+          console.log('Invalid credentials');
+          props.signInAttempt(true);
+        } else {
+          console.log('Issue with Authentication server');
+          props.signInAttempt(true);
+        }
+      });
```

Comment:
Were these changes linted to this way? just curious cuz I don't think my eslint did this for my axios post request

Reply:
Yea it did. Hmm yea the linter works in mysterious ways lol